api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java

```diff
@@ -122,64 +135,58 @@ public void findParametersWithEmptyDomainIds() throws Exception {
 
     @Test
     public void getSubQuery() throws Exception {
+        String TABLE_PREFIX = workbenchConfig.bigquery.projectId + "." + workbenchConfig.bigquery.dataSetId;
```

Comment:
Nit: only constants should be in all caps. You could probably have a getTablePrefix() helper method that returns this, since it's repeated.